build-system/pr-check/checks.js

```diff
@@ -81,6 +81,10 @@ async function prBuildWorkflow() {
     timedExecOrDie('amp ava');
   }
 
+  if (buildTargetsInclude(Targets.BUILD_SYSTEM)) {
+    timedExecOrDie('amp check-build-system');
```

Comment:
It will also need to be added to `tasks/pr-check.js` so it runs as part of `amp pr-check`, which is used during local development. Perhaps add it around line 77 below?

https://github.com/ampproject/amphtml/blob/6a05883aa72ee068e9062765deb8da67768cddcf/build-system/tasks/pr-check.js#L74-L80